### PR TITLE
Kernel: Remove modules

### PR DESCRIPTION
**Pull request by jcvenegas**
Remove modules from default kernel config.

Modules are not used in default kata images.

Lets remove them.

Fixes: #276

### CODE REVIEW

File: kernel/configs/x86_64_kata_kvm_4.14.x [hunk 1]
```diff
@@ -138,8 +138,9 @@ CONFIG_RCU_NEED_SEGCBLIST=y
 CONFIG_CONTEXT_TRACKING=y
 # CONFIG_CONTEXT_TRACKING_FORCE is not set
 CONFIG_RCU_NOCB_CPU=y
-# CONFIG_BUILD_BIN2C is not set
-# CONFIG_IKCONFIG is not set
+CONFIG_BUILD_BIN2C=y
+CONFIG_IKCONFIG=y
```
Comment (amshinde) [inline kernel/configs/x86_64_kata_kvm_4.14.x]: You have enabled the above options. If that was the intention, these should go in a separate commit with an explanation for the options.

Comment (jcvenegas) [reply kernel/configs/x86_64_kata_kvm_4.14.x]: seems that was enabled accidentally after the oldconfig, removed 👍

**Comment by jodh-intel** [comment]
lgtm

What about the ARM modules though?

@Pennyzct, @Weichen81 - do you want to look at removing those?

**Comment by Pennyzct** [comment]
Let's remove them ;)

**Comment by Weichen81** [comment]
Ok, these modules are never used for Arm64, let’s remove it.
